plugin/eqdetect/source/dextool/plugin/eqdetect/runner.d

```diff
@@ -57,9 +57,12 @@ ExitStatusType runPlugin(string[] args) {
 
         exit_status = analyzeFile(AbsolutePath(FileName(m.path)), cflags, visitor, ctx);
         import std.path : baseName;
-        import dextool.plugin.eqdetect.subfolder : writeToFile;
-        writeToFile(visitor.generatedSource.render, baseName(m.path), m.kind, m.id, false);
-        writeToFile(visitor.generatedMutation.render, baseName(m.path), m.kind, m.id, true);
+        import dextool.plugin.eqdetect.subfolder : writeToFile, SnippetFinder;
+        string source_path = writeToFile(visitor.generatedSource.render, baseName(m.path), m.kind, m.id, "_source_");
+        string mutant_path = writeToFile(visitor.generatedMutation.render, baseName(m.path), m.kind, m.id, "_mutant_");
+        auto s = SnippetFinder.generateKlee(visitor.function_params, source_path,
+        mutant_path, visitor.function_name);
+        writeToFile(s, baseName(m.path), m.kind, m.id, "_klee_");
         writeln("---------------------------------------");
```

Comment:
tip: use the logging framework. It makes it easier to control how much info is printed.

Reply:
Yes, the plugin are writing a lot of information that needs to be changed. Thank you for tip!